installing/installing_ibm_z/installing-ibm-z.adoc

```diff
@@ -49,6 +49,10 @@ include::modules/csr-management.adoc[leveloffset=+2]
 
 * See link:http://public.dhe.ibm.com/software/dw/linux390/perf/zvm_hpav00.pdf[Scaling HyperPAV alias devices on Linux guests on z/VM] for performance optimization.
 
+* See link:https://www.vm.ibm.com/library/presentations/lparperf.pdf[Topics in LPAR performance] for LPAR weight management and entitlements.
```

Comment:
In the original cherry-pick, the next line was:
```
* xref:../../scalability_and_performance/ibm-z-recommended-host-practices.adoc#ibm-z-recommended-host-practices[Recommended host practices for IBM Z & LinuxONE environments]
```

This line was breaking the build because the target file was only added to 4.7+ on https://github.com/openshift/openshift-docs/pull/35822.